package.json

```diff
@@ -46,7 +46,7 @@
     "@openzeppelin/contracts-upgradeable": "^4.8.3",
     "@openzeppelin/hardhat-upgrades": "^1.21.0",
     "@solidity-parser/parser": "^0.13.2",
-    "@venusprotocol/solidity-utilities": "^1.3.0",
+    "@venusprotocol/solidity-utilities": "^1.3.1-dev.1",
```

Comment:
We should avoid `-dev` versions if possible. The next version of the `solidity-utilities` package is 2.0.0, using solidity 0.8.25. If we use that package, maybe we need to merge also https://github.com/VenusProtocol/isolated-pools/pull/381 in this repo, rebase this PR and redeploy the contracts using 0.8.25